### ISSUE TITLE
YGGTorrent : Unable to bdecode torrent

### ISSUE THREAD

**Issue by Naokor** [pymedusa/Medusa#4723]
**Describe the bug**
The YggTorrent provider seems to have a bug. The downloads are not correctly handled to rtorrent. The "unable to bdecode torrent" occurs. Thank you for your help. 

**Medusa (please complete the following information):**

Branch: master 
Commit: e7c4ff3bc9a48b47c932b0e40a539d62e3a62f64 Version: v0.2.6 Database: 44.11
Python Version: | 2.7.12 (default, Dec 4 2017, 14:50:18) [GCC 5.4.0 20160609]
SSL Version: | OpenSSL 1.0.2g 1 Mar 2016
OS: | Linux-4.12.14-041214-generic-x86_64-with-Ubuntu-16.04-xenial

**Logs:**
<details>

```
2018-07-23 12:45:32 WARNING  SEARCHQUEUE-FORCED-325517 :: [e7c4ff3] Unable to bdecode torrent. Invalid torrent: Salvation.S02E01.FASTSUB.VOSTFR.720p.AMZN.WEB-DL.DDP5.1.H.264-GOLD. Deleting cached result if exists
2018-07-23 12:45:32 INFO     SEARCHQUEUE-FORCED-325517 :: [e7c4ff3] Downloading Salvation.S02E01.FASTSUB.VOSTFR.720p.AMZN.WEB-DL.DDP5.1.H.264-GOLD with 36 seeders and 0 leechers and size 938.52 MB from Yggtorrent
2018-07-23 12:45:32 INFO     SEARCHQUEUE-FORCED-325517 :: [Yggtorrent] :: [e7c4ff3] Picked Salvation.S02E01.FASTSUB.VOSTFR.720p.AMZN.WEB-DL.DDP5.1.H.264-GOLD as the best result.
2018-07-23 12:45:32 INFO     SEARCHQUEUE-FORCED-325517 :: [Yggtorrent] :: [e7c4ff3] Quality of Salvation.S02E01.(FASTSUB.VOSTFR.720p.WEBRip.x265.NSP) is 720p WEB-DL
2018-07-23 12:45:32 INFO     SEARCHQUEUE-FORCED-325517 :: [Yggtorrent] :: [e7c4ff3] Quality of Salvation.S02E01.FASTSUB.VOSTFR.720p.HDTV.x264.-.ZKH is 720p HDTV
2018-07-23 12:45:32 INFO     SEARCHQUEUE-FORCED-325517 :: [Yggtorrent] :: [e7c4ff3] Quality of Salvation.S02E01.FASTSUB.VOSTFR.720p.AMZN.WEB-DL.DDP5.1.H.264-GOLD is 720p WEB-DL
2018-07-23 12:45:31 INFO     SEARCHQUEUE-FORCED-325517 :: [Yggtorrent] :: [e7c4ff3] Performing episode search for Salvation```
</details>

**Comment by Naokor** [comment]
Hello,

@medariox I tried with the develop file and there is still something wrong apparently : 

Rutorrent said : 
Command on torrent creation failed: Junk at end of input.

If it is "normal" ignore this message :)

Thank you!

**Comment by p0psicles** [comment]
Does rtorrent receive the torrent?

**Comment by Naokor** [comment]
@p0psicles Yes indeed! The problem probably not on Medusa's side...

**Comment by medariox** [comment]
Great to hear! Closing as fixed.

**Comment by 29z** [comment]
Hi, I was just wondering if there an alternative work around for this? 

I've got other apps that use rtorrent(not yggtorrent) with and I am able to specify the download location and everything works(mylar, (sickrage-switching from) etc). 

When using Medusa on my seedbox, I get torrents added to rtorrent but with an error "Junk at end of file". If I leave it blank like mentioned above it does get added to rtorrent without any errors, however it places it in my default directory which messes with my local medusa for post processing.  As I've just found out that its been deleting my music DLs when i set the post-processing dir to my default local directory that everything is synced to, I'd love to be able to set it properly. I've tried all 100% accurate directories in medusa remote, all resulting in junk at end of file, even when browsing the directory i want through the browse button still resulting in the same error. I've been using medusa locally for like 2 years and sickrage remotely untill switching to medusa remotely as well, So If anyone knows a trick to this I'd be greatly appreciative. 

Thanks
